deploy-osp-adoption.yml

```diff
@@ -0,0 +1,93 @@
+---
+# Purpose of this playbook:
+# This playbook deploy OSP 17.1 on previously created
+# infra for adoption. The main goal of this playbook
+# is to be called as a standalone piece of a bigger
+# job, mostly in the adoption context.
+#
+# You would typically run this playbook after the
+# "create-infra.yml"
+
+- name: Parent scenario if needed
+  hosts: "{{ cifmw_target_host | default('localhost')  }}"
+  gather_facts: true
+  vars:
+    cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data"
+    _adoption_source_scenario_file: >-
+      {{
+        [cifmw_adoption_source_scenario_path,
+         cifmw_architecture_scenario ~ '.yaml'] | path_join
+      }}
+  tasks:
+    - name: Inherit from parent scenarios if needed
+      ansible.builtin.include_tasks:
+        file: "ci/playbooks/tasks/inherit_parent_scenario.yml"
+
+    - name: Ensure networking data is loaded
+      ansible.builtin.import_role:
+        name: networking_mapper
+        tasks_from: load_env_definition.yml
+
+    - name: Load source adoption scenario
+      ansible.builtin.include_vars:
+        file: "{{ _adoption_source_scenario_file }}"
+        name: _adoption_source_scenario
+
+    - name: Read inventory groups
+      ansible.builtin.slurp:
+        path: "/home/zuul/ci-framework-data/reproducer-inventory/all-group.yml"
```

Comment:
cifmw_basedir var should be used instead of this hard-coded value

Reply:
Done, thanks